eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -44,4 +47,28 @@ catch
   Write-Host $_.Exception.Message
 }
 
-return $releaseNotes
\ No newline at end of file
+if ([System.String]::IsNullOrEmpty($VersionString)) 
+{
+  return $releaseNotes
+}
+else 
+{
+  if ($releaseNotes.ContainsKey($VersionString)) 
+  {
+    $releaseNotesForVersion = $releaseNotes[$VersionString].ReleaseContent
+    $releaseNotesArray = $releaseNotesForVersion.Split([Environment]::NewLine)
+    $processedNotes = New-Object System.Collections.ArrayList
+    foreach ($line in $releaseNotesArray) 
```

Comment:
The more powershell way to do this would be something like:
```
$procceseedNotes = $relaeseNotesForVersion -split [Environment]::NewLine | where { $_ -notmatch $RELEASE_TITLE_REGEX }

return $processedNotes -Join [Environment]::NewLine
```

You can probably also eliminate the need to split on newline if you make the release notes property in the dictionary just be a set of lines as I see we are joining that together above as well.

Reply:
Actually looking at the code above it doesn't look like we are including the version header line in the output so you shouldn't need to do any of this work. Just return ` $releaseNotes[$VersionString].ReleaseContent` directly. Or am I missing something?

Reply:
The ReleaseContent also contains the header with the release marker(_e.g. ## 1.0.2preview.1 (2020-02-28)). The code add the header to the contentArrays, I just felt it would be redundant to include that again on Nuget which is why am removing it. I did not want to change the existing returned value at this time so as not to break other part of the Git Tagging and release script.

Reply:
Thanks. Updated this with a [Pull Request](https://github.com/Azure/azure-sdk-tools/pull/406)